lib/vmpooler/pool_manager.rb

```diff
@@ -565,7 +571,10 @@ def _check_pool(pool)
       total = $redis.scard('vmpooler__pending__' + pool['name']) + ready
 
       begin
-        if defined? $graphite
+        if defined? $statsd
```

Comment:
You don't need to use `defined?` for any of these globals because an uninitialized global variable should still return `nil` when it's called:

```ruby
# Ruby 2.1.10
>> RUBY_VERSION
=> "2.1.10"
>> RUBY_
RUBY_COPYRIGHT     RUBY_ENGINE        RUBY_PLATFORM      RUBY_REVISION
RUBY_DESCRIPTION   RUBY_PATCHLEVEL    RUBY_RELEASE_DATE  RUBY_VERSION
>> RUBY_ENGINE
=> "ruby"
>> $amidefined
=> nil
>> $noiamnot
=> nil
>>

# Ruby 2.3.1
>> RUBY_VERSION
=> "2.3.1"
>> RUBY_ENGINE
=> "ruby"
>> $amidefined
=> nil
>> $noiamnot
=> nil
>>

# Jruby 1.7.25
>> RUBY_VERSION
=> "1.9.3"
>> RUBY_ENGINE
=> "jruby"
>> $amidefined
=> nil
>> $noiamnot
=> nil
>>
```